regression/esbmc-solidity/doftcoin_1/contract.sol

```diff
@@ -0,0 +1,192 @@
+/**
+ *Submitted for verification at Etherscan.io on 2017-09-28
+*/
+
+
+
+
+// do not add pragma statement here, it will cause problems with the solcmc
+
+// pragma solidity ^0.8.18;
+contract owned { 
+    address public owner;
+    
+    constructor() {
+        owner = msg.sender;
+    }
+
+    function transferOwnership(address newOwner) public {
+        owner = newOwner;
+    }
+}
+
```

Comment:
The regression tests are a bit of a mess :-) Can you remove empty lines, comments, etc?